internalstaging.theanvil.io/manifest.json

```diff
@@ -107,6 +107,159 @@
         }
       ],
       "restart_policy": "Never"
+    },
+    {
+      "name": "ingest-metadata-manifest",
+      "action": "ingest-metadata-manifest",
+      "activeDeadlineSeconds": 86400,
+      "serviceAccountName": "sower-jobs-internalstaging-theanvil-io-sa",
```

Comment:
I think the default SA is just `jobs-{{env}}` instead of `sower-jobs-{{env}}` but this might be lingering from a version before?

Reply:
Yea it was setup on an older version so it seems to match what is in eks.